src/app/inventory/store/d1-item-factory.ts

```diff
@@ -497,7 +497,7 @@ function buildTalentGrid(
     }
 
     // Only one node in this column can be selected (scopes, etc)
-    const exclusiveInColumn = Boolean(talentNodeGroup.exlusiveWithNodes?.length);
+    const exclusiveInColumn = Boolean(talentNodeGroup.exclusiveWithNodes?.length);
```

Comment:
This may be worth verifying...

Reply:
That one's an actual typo on Bungie's side